client/src/components/composite/SignUpForm/SignUpForm.story.tsx

```diff
@@ -77,7 +77,7 @@ export const DefaultStory = () => {
           }
         },
         {
-          title: "last",
+          title: "",
```

Comment:
This is causing problems, you should set the title here

Reply:
![image](https://github.com/UoaWDCC/uasc-web/assets/64724157/8723079a-ab3e-4fe2-adae-542705726ee1)
- desktop version